client/signup/steps/site-information/index.jsx

```diff
@@ -0,0 +1,157 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import i18n, { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import { isUserLoggedIn } from 'state/current-user/selectors';
+import { getSiteInformation } from 'state/signup/steps/site-information/selectors';
+import { setSiteInformation } from 'state/signup/steps/site-information/actions';
+import { getSiteTitle } from 'state/signup/steps/site-title/selectors';
+import { setSiteTitle } from 'state/signup/steps/site-title/actions';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+import FormLabel from 'components/forms/form-label';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+
+class SiteInformation extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			name: props.siteTitle,
+			address: props.siteInformation.address,
+			phone: props.siteInformation.phone,
+		};
+	}
+
+	handleInputChange = ( { target: { name, value } } ) => this.setState( { [ name ]: [ value ] } );
+
+	handleSubmit = event => {
+		event.preventDefault();
+		this.props.submitStep( ...this.state );
+	};
+
+	renderContent() {
+		const { translate } = this.props;
+
+		return (
+			<div className="site-information__wrapper">
+				<div className="site-information__form-wrapper ">
+					<form onSubmit={ this.handleSubmit }>
+						<Card>
+							<h3>{ translate( 'Enter your basic information' ) }</h3>
+
+							<FormFieldset>
+								<FormLabel htmlFor="name">
+									{ translate( 'Business Name' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Business name' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).